### DISCUSSION TITLE
Why does localsend ambiently use 3% GPU on Windows 10?

### DISCUSSION THREAD

**Discussion by calvinusesyourcode** [localsend/localsend#1083] [Discussion (English, International)] [answered]
Installed from official source. The 3% ambient GPU - 3D utilization is only localsend, and then dropped when I open the window from system tray...

Offshore bitcoin mining on my PC?

**Comment by Tienisto** [comment, marked as answer]
The spinning logo uses some GPU. You can either go to the settings tab or you can use the "minimize to tray" feature.

Funny that you assume that 3% GPU can mine some Bitcoin. I think you need to wait decades because the probability to find one is near zero. Usually, you need to buy special hardware to compete in the current mining environment

**Reply by calvinusesyourcode** [reply]
this is silly though when I'm tryna use CUDA for other things

I've never contributed to someone else's project so it would be cool if you give me a good starting file to look at and I'll submit a PR to add an option in advanced options to disable the spinning render :D  

EDIT: because "minimize to tray" on windows **does not** stop the 3% GPU rendering
